web_monitoring/cli.py

```diff
@@ -0,0 +1,67 @@
+# Command Line Interface
+# See scripts/ directory for associated executable(s). All of the interesting
+# functionality is implemented in this module to make it easier to test.
+from docopt import docopt
+from tqdm import tqdm
+from web_monitoring import internetarchive as ia
+from web_monitoring import pf_edgi as pf
+from web_monitoring import db
+
+
+# These functions lump together library code into monolithic operations for the
+# CLI. They also print. To access this functionality programmatically, it is
+# better to use the underlying library code.
+
+
+def import_ia(url, agency, site):
+    print('obtaining versions list from Internet Archive...')
+    versions = ia.list_versions(url)
+    # Collect all the results and POST them as a unit (rather than streaming).
+    formatted_versions = []
+    for dt, uri in tqdm(versions, desc='formatting', unit='versions'):
```

Comment:
formatting: 34versions [00:02, 10.49versions/s]

The lack of space between the number and “versions” is really bugging me here :P